dockers/docker-snmp/snmpd.conf.j2

```diff
@@ -44,6 +44,34 @@ rocommunity {{ snmp_rocommunity }}
 rocommunity6 {{ snmp_rocommunity }}
 {% endif %}
 
+{% if SNMP_COMMUNITY is defined %}
+{% for community in SNMP_COMMUNITY.keys() %}
```

Comment:
>.keys() [](start = 34, length = 7)

You can remove `.keys()`, as in python3.

Reply:
fixed.